content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -422,14 +422,14 @@ An `int` option requires a range attribute to specify the minimum and maximum ra
      
     [[range(3, 16)]] // This integer option accepts values between 3 and 16 (both ends included).  
     option int o_numberOfTaps;
-  
-Even though the Shader Variant Options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
-To learn more about how Shader Variant Options are encoded when compiled, please read: [Shader Variant Options & The Fallback Key.](shader-variants-fallback-key)  
+```
+Even though the Shader variant options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
+To learn more about how Shader variant options are encoded when compiled, please read: [Shader variant options & The Fallback Key.](shader-variants-fallback-key)  
   
 # <a name="padton"></a>The special attribute [[pad_to(N)]]
```

Comment:
Fix header as described previously.

Reply:
What would be the encoding for this anchor?
"# The special attribute \[\[pad_to(N)\]\]

Reply:
#the-special-attribute-pad_ton

In case you're wondering how to find this out, I have a few tools that make it easy. 
1. The **Markdown All in One** extension for VS Code. It has an autocomplete feature and told me the link: 
![image](https://user-images.githubusercontent.com/75444793/144963021-bfc23540-750b-4979-b69b-b0c6a229d881.png)
2. When I preview the website on my local machine (requires that you install Hugo), I navigate to the anchor icon by the section name, and click on the URL. The encoding is written in the browser URL: 
![image](https://user-images.githubusercontent.com/75444793/144963285-66bb4b33-df41-47ab-a10d-75771a0e388d.png)